conjur/api/client.py

```diff
@@ -14,15 +14,16 @@
 import netrc
 
 from Utils.utils import Utils
-from conjur.api import Api
+from conjur.api.api import Api
```

Comment:
hmmm is there any way to change one of the api.api?
looks a bit strange...

Reply:
I moved it to init file so in here it will be 
`from conjur.api import api`